src/api/stops/StopsAPI.test.js

```diff
@@ -0,0 +1,59 @@
+    );
+
+    return api
+      .search({
+        q: 'Bern',
+      })
+      .then((features) => {
+        // Correct url
+        expect(fetch.mock.calls[0][0]).toEqual(
+          'https://api.geops.io/stops/v1/?key=apiKey&q=Bern',
+        );
+
+        // Correct search result
+        expect(features[0].properties.name).toEqual('Bern');
+      });
+  });
+
+  test('search on error.', () => {
+    // Mock console statement
+    const consoleOutput = [];
+    // eslint-disable-next-line no-console
+    console.warn = (message, err) =>
+      consoleOutput.push([message, err.toString()]);
+
+    fetch.mockResponseOnce('invalid json');
```

Comment:
Currently, in case of 
`'{"error": "Authorization field missing"}'`
nothing happens. 

@oterral Should we also handle this kind of error?

Reply:
i would say yes